src/util/object_forwarding.rs

```diff
@@ -74,12 +74,30 @@ pub fn spin_and_get_forwarded_object<VM: VMBinding>(
     }
 }
 
+/// Copy an object and set the forwarding state.
+///
+/// The caller can use `on_after_forwarding` to set extra metadata (including VO bits, mark bits,
+/// etc.) after the object is copied, but before the forwarding state is changed to `FORWARDED`. The
+/// atomic memory operation that sets the forwarding bits to `FORWARDED` has the `SeqCst` order.  It
```

Comment:
Actually minor point. This is an implementation detail, so maybe we shouldn't document it in the API doc comment

Reply:
I think it's fine. This function should be private to the crate, not public.

Reply:
That's OK.  The entire `util::object_forwarding` module is private, so the doc comments are only relevant to the developers of the mmtk-core.  We may update the doc comments if we change it to acquire-release later.

Reply:
Sure 👍